app/app.go

```diff
@@ -92,9 +112,16 @@ func homeHandler(c *gin.Context, settings common.AppSettings, db database.Databa
 
 	err = index_view.Render(c, html_buffer)
 	if err != nil {
-		log.Error().Msgf("could not render index: %v", err)
+		log.Error().Msgf("Could not render index: %v", err)
 		return []byte{}, err
 	}
 
 	return html_buffer.Bytes(), nil
 }
+
+func max(a, b int) int {
```

Comment:
There's already a function in the standard called `max(...)` examples [here](https://www.linkedin.com/pulse/built-in-min-max-functions-go-121-robert-gogolok/)